SFR/outer_fuel_pincell/make_SFR_pincell_model.py

```diff
@@ -0,0 +1,113 @@
+import openmc
+import numpy as np
+from argparse import ArgumentParser
+import openmc.material
+from materials import *
+import common_input as geom
+
+
+def settings(shannon_entropy: bool, height: float):
+    pitch = geom.pitch
```

Comment:
If you are using `pitch` directly from `geom`, why not use `height` as well, instead of passing it as an argument?